### ISSUE TITLE
Invoice editing causes line items to save incorreclty

### ISSUE THREAD

**Issue by nemerald-voip** [akaunting/akaunting#2945]
### Akaunting version

3.0.12

### PHP version

8.1.12

### Operating system

Debian 11

### Steps to reproduce

When the invoice is first added and saved with two items 
1) Item 1 with ID 100
2) Item 2 with ID 200

And then, the invoice is edited, and another item 3, with ID 300, is added and moved to the 1st position, the IDs aren't saved correctly in the tw2_document_items table. They get switched. 

### Expected result

Here is the expected order and their respectful IDs:
1) Item 3 with ID 300
2) Item 1 with ID 100
3) Item 2 with ID 200

### Actual result

In the database table, it's saved as follows. 
1) Name and description, price, etc., is for Item 1, but ID shows as ID 300
2) Name and description, price, etc., is for Item 2, but ID shows as ID 100
3) Name and description, price, etc., is for  Item 3, but ID shows as ID 200

So basically IDs and saved Items don't match anymore. 

### Additional comments

I will need to do some more testing, but I think it only happens when the added item is put in the first position

**Comment by cuneytsenturk** [comment]
Thansk your feedback.
I've tested clean Akaunting, everything works so then please wait published new release and update it then test again.
